leonardo-lfonzi62/week-1/mockup-html-css-lab/starter-code/css/main.css

```diff
@@ -0,0 +1,82 @@
+* {
+  font-family: lato;
+}
+
+
+h1{
+   color: #dfc12a;
+   font-size: 40px;
+}
+
+a {
+  text-decoration: none;
+  color: #dfc12a;
+}
+
+nav a {
+  margin: 8px;
+  text-decoration: underline;
+  text-underline-position: under;
+  color: #D3D3D3;
+  /*Using Border property instead of | in HTML to create space btw the nav items*/
+  /*border-right: 1px solid black ;*/
```

Comment:
Commenting like this is immensely helpful for code updates and explanations of what's going on and what has changed - keep it up, Leo!